server/datastore/mysql/hosts.go

```diff
@@ -816,7 +816,7 @@ func (d *Datastore) SearchHosts(ctx context.Context, filter fleet.TeamFilter, qu
 	args = append(args, in)
 	sqlb.WriteString(" id NOT IN (?) AND ")
 	sqlb.WriteString(d.whereFilterHostsByTeams(filter, "h"))
-	sqlb.WriteString(` ORDER BY COALESCE(hst.seen_time, h.created_at) DESC LIMIT 10`)
+	sqlb.WriteString(` ORDER BY h.id DESC LIMIT 10`)
```

Comment:
Let's add a sort check to a test to make sure we don't change this in the future.

Even with `LIMIT 10` this was causing performance issues?

Reply:
I guess it has to sort all first :)

Reply:
Right, it needs to scan it all in order to find the first 10 that match the ordering defined.